articles/2023-06-15-renovate.md

```diff
@@ -440,12 +439,12 @@ repos:
         #   )
 ```
 
-And automate `pre-commit` executions in PRs like [this](https://github.com/antonbabenko/pre-commit-terraform/blob/32597545e7fff922a19c3b3c7cecf3fd6bc6b628/.github/workflows/pre-commit.yaml) or [that](https://github.com/SpotOnInc/renovate-config/blob/dfcbbf82cecc20e6d759c058399fdfda3956c62f/.github/workflows/pre-commit.yaml) or in any other way.
+And automate `pre-commit` executions in PRs like [this](https://github.com/antonbabenko/pre-commit-terraform/blob/32597545e7fff922a19c3b3c7cecf3fd6bc6b628/.github/workflows/pre-commit.yaml) or [this](https://github.com/SpotOnInc/renovate-config/blob/dfcbbf82cecc20e6d759c058399fdfda3956c62f/.github/workflows/pre-commit.yaml).
 
 ## Summary
 
-Here are a few points:
+Here are a few takeaways:
 
-* Better have lockifles than have no (repeatability, security)
-* Better automate all these updates to maintain the same versions across codebase and be on the cutting edge without bleeding ([Renovate](https://github.com/renovatebot/renovate), [dependabot](https://github.com/dependabot/dependabot-core), [tfupdate via pre-commit](https://github.com/antonbabenko/pre-commit-terraform/#tfupdate))
-* For a better experience, you need a source of truth (automated `terraform plan` in CI/Terratest etc.) which will show that made changes did not break anything (it can be [Atlantis](https://github.com/runatlantis/atlantis), [Spacelift](https://spacelift.io/), [Terraform Cloud](https://www.hashicorp.com/products/terraform) or you can do it in your own CI)
+* It's better to have lockifles than have no (repeatability, security)
+* It makes sense to automate all these updates and maintain the same versions across the codebase, and be on the cutting edge without bleeding ([Renovate](https://github.com/renovatebot/renovate), [dependabot](https://github.com/dependabot/dependabot-core), [tfupdate via pre-commit](https://github.com/antonbabenko/pre-commit-terraform/#tfupdate))
+* For a better user experience, you need a source of truth (automated `terraform plan` in CI/Terratest, etc.) which shows that changes do not break anything. It can be [Atlantis](https://github.com/runatlantis/atlantis), [Spacelift](https://spacelift.io/), [Terraform Cloud](https://www.hashicorp.com/products/terraform), or you can do it in your own CI.
```

Comment:
```suggestion
* For a better user experience, you need a source of truth (automated `terraform plan` in CI, Terratests, etc.) which shows that changes do not break anything. It can be [Atlantis](https://github.com/runatlantis/atlantis), [Spacelift](https://spacelift.io/), [Terraform Cloud](https://www.hashicorp.com/products/terraform), or you can do it in your own CI.
```